imagefit/templatetags/imagefit.py

```diff
@@ -34,3 +34,11 @@ def static_resize(value, size):
     return string url
     """
     return resize(value, size, 'static_resize')
+
+@register.filter
+def external_resize(value, size):
+    """
+    Generates the url for the resized image for external image
+    return string url
+    """
+    return resize(value, size, 'external_resize')
```

Comment:
I'm aware I'm the one who suggested a new template filter to manage external resources.
However do you think we could keep the same filter and have below `if path_name.startswith(('http://', 'https://'))` or is there something I'm not seeing?

Reply:
I think seperate template filter is more managable